Grammar/python.gram

```diff
@@ -218,15 +218,17 @@ import_from_targets[asdl_alias_seq*]:
 import_from_as_names[asdl_alias_seq*]:
     | a[asdl_alias_seq*]=','.import_from_as_name+ { a }
 import_from_as_name[alias_ty]:
-    | a=NAME b=['as' z=NAME { z }] { _PyAST_alias(a->v.Name.id,
-                                               (b) ? ((expr_ty) b)->v.Name.id : NULL,
-                                               EXTRA) }
+    | a=NAME 'as' b=NAME &(',' | ')' | NEWLINE) { 
+        _PyAST_alias(a->v.Name.id, b->v.Name.id, EXTRA) }
+    | invalid_import_from_as_name
+    | a=NAME { _PyAST_alias(a->v.Name.id, NULL, EXTRA) }
```

Comment:
Same here, I am not too fond of changing working rules when introducing new invalid ones